ui/src/types/status-types-and-constants.ts

```diff
@@ -56,6 +64,17 @@ export const STATUS_BACKGROUND_COLOR: {
   FINISHED_WITH_ISSUES: 'bg-yellow-500',
 } as const;
 
+// Map Vulnerability ratings to TailwindCSS background colors
+export const VULNERABILITY_RATING_BG_COLOR: {
+  [K in VulnerabilityRating]: string;
+} = {
+  CRITICAL: 'bg-red-600',
+  HIGH: 'bg-orange-600',
+  MEDIUM: 'bg-amber-500',
+  LOW: 'bg-yellow-400',
+  NONE: 'bg-neutral-300',
+} as const;
```

Comment:
As they were added in this PR, I'd actually move them to the function file now, so there's only the one way to get these through the function.